ReactApp/src/components/EditorToolbar.jsx

```diff
@@ -20,7 +20,7 @@ const EditorToolbar = (props) => {
 		addBlockButton = <Inserter />;
 	} else {
 		addBlockButton = (
-			<button onClick={() => postMessage('showBlockPicker')}>+</button>
+			<button onClick={() => showBlockPicker() }>+</button>
```

Comment:
We might remove the unnecessary anonymous function to improve readability. Feel free to disagree. 
```suggestion
			<button onClick={ showBlockPicker }>+</button>
```